server/core/votes-manager.js

```diff
@@ -29,6 +47,10 @@ module.exports = class VotesManager {
 	}
 
 	getVotesCount(song) {
+		if(this.partyMode) {
+			return parseInt(Math.random() * 100);

```

Comment:
Weird thing is that the votes on the currently played song are out of control (way above 100). 😄 
Let's say it's caused by all the partyness.